wrapper/fmriprep_docker.py

```diff
@@ -371,11 +373,12 @@ def main():
                'DOCKER_VERSION_8395080871=%s' % docker_version]
 
     # Patch working repositories into installed package directories
-    for pkg in ('fmriprep', 'niworkflows', 'nipype'):
-        repo_path = getattr(opts, 'patch_' + pkg)
-        if repo_path is not None:
-            command.extend(['-v',
-                            '{}:{}/{}:ro'.format(repo_path, PKG_PATH, pkg)])
+    if opts.patch:
+        for pkg, repo_path in opts.patch.items():
+            if repo_path is not None:
```

Comment:
Is it possible for `repo_path` to be `None`?

Reply:
i don't think so, but wanted to cover any bases i missed. let's remove it and see what happens